core/camel-core/src/test/java/org/apache/camel/language/TokenXMLPairNamespaceSplitTest.java

```diff
@@ -66,7 +66,7 @@ public void testTokenXMLPair2() throws Exception {
     protected String createBody() {
         StringBuilder sb = new StringBuilder("<?xml version=\"1.0\"?>\n");
         sb.append("<orders xmlns=\"http:acme.com\">\n");
-        sb.append("  <order id=\"1\">Camel in Action</order>\n");
+        sb.append("  <order id=\"1\" xmlns=\"http:acme.com\">Camel in Action</order>\n");
```

Comment:
It would be better to duplicate the test and keep the original as-is and then in the new test check for inherited root namespace so we test both conditions